src/screens/CreateWalletScreen.tsx

```diff
@@ -32,38 +32,28 @@ interface Props {
 }
 interface State {
   label: string;
-  selectedIndex: number;
+  selectedType: WalletType;
```

Comment:
Until now, it wasn't possible to change the order of rendering the radio buttons just like that, without making changes in other places. Thanks to replacing `selectedIndex` with `selectedType` we should be possible to do that